app/views/devise/shared/_error_messages.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if resource.errors.any? %>
+  <div id="error_explanation">
+    <h2>
+      <%= I18n.t("errors.messages.not_saved",
+                 count: resource.errors.count,
+                 resource: resource.class.model_name.human.downcase)
+       %>
```

Comment:
can't you just use 1 line instead of 3?

Reply:
we will not make the change, the line would be very long 🖏